contentcuration/contentcuration/frontend/channelEdit/components/AssessmentEditor/AssessmentEditor.spec.js

```diff
@@ -334,62 +350,73 @@ describe('AssessmentEditor', () => {
         order: 2,
         isNew: true,
       });
+      expect(listeners.addItem).toBeCalledTimes(1);
     });
 
     it('emits update item events with updated order of items below the new item', () => {
```

Comment:
I'd suggest updating test descriptions to reflect the new logic. For example here, it might be `emits update items event with new orders` or something similar?